api/resources_portal/models/organization_invitation.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+
+from resources_portal.models.organization import Organization
+from resources_portal.models.user import User
+
+
+class OrganizationInvitation(models.Model):
+    """ This model will store information on invitations to join an organization and requests to join an organization """
+
+    class Meta:
+        db_table = "organization_invitations"
+        get_latest_by = "created_at"
```

Comment:
Why created_at instead of updated_at?

Reply:
I suppose updated_at makes more sense. Thanks!